### PR TITLE
Add vagrant config

### PR DESCRIPTION
**Pull request by phoeinx**

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+```bash
+systemctl restart mcs-<process_name>
+```
+
+### Zombie/Defunct processes
+
+Sometimes Zombie/Defunct processes can cause problems, e.g. when one PrimProc process has become defunct, a new one has been started, but MariaDB still attaches to the defunct process. Refer to [here](https://www.linuxjournal.com/content/how-kill-zombie-processes-linux) for ways to kill Zombie processes. Sometimes a restart of the development machine can be the quickest way to resolve this problem, however.
+
+## Development Setup with Local Linux VM for M1 Macs
```
Comment (drrtuy) [inline DEVELOPING.md]:
```suggestion
## Development Setup with Local Linux VM for ARM-based Macs
```

**Review by drrtuy — APPROVED** [review]
LGTM except a tiny change.

File: DEVELOPING.md (hunk 1 above)
Comment (phoeinx) [reply DEVELOPING.md]: Good point! I adapted it.

**Comment by phoeinx** [comment]
Close to open against MCS repo.

**Pull request closed without merging**